app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/declaration/Declaration.scala

```diff
@@ -59,9 +59,17 @@ object PriceOfGoods {
   implicit val format: OFormat[PriceOfGoods] = Json.format[PriceOfGoods]
 }
 
-case class GoodsEntry(typeOfGoods: String,
+case class CategoryQuantityOfGoods(category: String, quantity: String)
+
+object CategoryQuantityOfGoods {
+  implicit val format: OFormat[CategoryQuantityOfGoods] = Json.format[CategoryQuantityOfGoods]
+}
+
+case class GoodsEntry(maybeCategoryQuantityOfGoods: Option[CategoryQuantityOfGoods] = None,
```

Comment:
I reckon will be the only non-optional element.

My reasoning is that you will only create a GoodsEntry after the user has submitted the /search-goods page (or whatever its called) and at that time you will have a CategoryQuantityOfGoods

Reply:
Okay, am happy to change to be non optional